packages/types/src/xlucene-interfaces.ts

```diff
@@ -0,0 +1,21 @@
+export enum XluceneFieldType {
+    Geo = 'geo',
+    Date = 'date',
+    IP = 'ip',
+    String = 'string',
+    Integer = 'integer',
+    Float = 'float',
+    Boolean = 'boolean',
+    Object = 'object',
+    GeoPoint = 'geo-point',
+    GeoJSON = 'geo-json',
+    Number = 'number'
+}
+
+export interface XluceneTypeConfig {
```

Comment:
It is xLucene not Xlucene

```suggestion
export interface xLuceneTypeConfig {
```